include/tcpdf_static.php

```diff
@@ -1136,7 +1136,7 @@ public static function extractCSSproperties($cssdata) {
 	 * @see setHtmlVSpace()
 	 * @public static
 	 */
-	public static function fixHTMLCode($html, $default_css='', $tagvs='', $tidy_options='', &$tagvspaces) {
+	public static function fixHTMLCode($html, $default_css, $tagvs, $tidy_options, &$tagvspaces) {
```

Comment:
https://stackoverflow.com/a/16636807/5155484
```suggestion
	public static function fixHTMLCode($html, $default_css = '', $tags = '', $tidy_options = '', &$tagvspaces = null) {
```